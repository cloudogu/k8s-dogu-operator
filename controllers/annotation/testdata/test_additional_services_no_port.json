{
  "Labels": {
    "SERVICE_TAGS": "webapp",
    "SERVICE_NAME": "admin"
  },
  "Env": [
    "SERVICE_ADDITIONAL_SERVICES=[{\"name\":\"admin-api\",\"location\":\"api\",\"pass\":\"admin/api/v2/\"}]"
  ],
  "ExposedPorts": {
    "80/tcp": {}
  }
}