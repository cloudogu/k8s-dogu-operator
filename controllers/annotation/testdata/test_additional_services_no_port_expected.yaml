# This yaml contains a ces service annotation for the web app. Only the annotations are compared.
apiVersion: v1
kind: Service
metadata:
  name: serviceName
  annotations:
    k8s-dogu-operator.cloudogu.com/ces-services: '[{"name":"admin","port":80,"location":"/admin","pass":"/admin"},{"name":"admin-api","port":80,"location":"/api","pass":"/admin/api/v2/"}]'
spec:
  selector:
    name: serviceName
  ports:
    - protocol: TCP
      port: 80